src/Pages/EmailList/email-template.css

```
@@ -0,0 +1,23 @@
+.email-template {
+    margin-top: 20px;
+    padding: 0 15px;
+    width: 100%;
+}
+
+#email-message {
+    border-radius: 6px;
+    background-color: white;
+    margin-top: 15px;
+    padding: 15px 25px;
+    height: 65vh;
+    overflow-y: auto;
+}
+#email-message p {
+    display: inline-block;
+}
+
+#copy-notification2 {
+    display: none;
+    padding-top: 10px;
+    color: rgb(2, 107, 126);
+}
```

add one blank line at the bottom of this file 
```suggestion
}

```